config/helm/appmesh-controller/ci/values.yaml

```diff
@@ -5,5 +5,5 @@ accountId: 123456789
 region: us-west-2
 image:
   repository: jeremyymill/appmesh-controller
```

Comment:
so this is a weird thing where the CI on eks-charts actually pulls from dockerhub instead of ECR; current workaround is to upload the image to dockerhub and put the repo name and tag here

Reply:
oh i see what you mean. ok so i should push to my dockerhub with the proper tag and change the repository value here?

Reply:
<img width="734" alt="Screen Shot 2022-09-07 at 12 38 23" src="https://user-images.githubusercontent.com/6402776/188962584-6e2533c5-cc9b-489e-a331-0342cde6e07c.png">